### ISSUE TITLE
sample: indent in multi line string

### ISSUE THREAD

**Issue by B4dM4n** [nix-community/nixpkgs-fmt#197]
Applying `nixpkgs-fmt` to the input twice yields the desired output.

Side note: input is formatted with `nixpkgs-fmt-0.7.0`.

## Input
```nix
let
  cfg = ''
    ${if true then ''
    bla
  '' else ''
    blub
  ''}
  '';
in
cfg
```

## Output
```nix
let
  cfg = ''
    ${
      if true then ''
      bla
    '' else ''
      blub
    ''}
  '';
in
cfg

```

## Desired output

```nix
let
  cfg = ''
    ${
      if true then ''
        bla
      '' else ''
        blub
      ''}
  '';
in
cfg
```

**Comment by Rizary** [comment]
@B4dM4n thank you! let me look into it.

**Comment by Rizary** [comment]
closed with #202 merged. 

Right now, if don't get expanded inside string interpolation
